mt-batcher/services/restorer/handle.go

```diff
@@ -148,6 +148,10 @@ func (s *DaService) GetDtlBatchTransactionByDataStoreId(c gecho.Context) error {
 		for i := 0; i < len(newBatchTxn); i++ {
 			l2Tx := new(types.Transaction)
 			txDecodeMetaData := new(eigenda.TransactionMeta)
+			if newBatchTxn[i].TxMeta == nil {
+				log.Error("Batch tx metadata shouldn't be nil")
+				continue
```

Comment:
Can we just skip this tx? Perhaps, we might need to return error and wait for developers to check this exception?

Reply:
We can't skip this transaction, because the batch submitter always rollup transactions sequentially, and currently it will block on that particular batch, keep retrying and returning an error, a metrics warning will be added here

Reply:
the error will return out directly